cli/tests/pcluster/cli/test_commands.py

```diff
@@ -333,3 +343,101 @@ def test_setup_bucket_with_resources_upload_failure(
         cluster._upload_instance_types_data()
 
     check_bucket_mock.assert_called_with()
+
+
+@pytest.mark.parametrize(
+    (
+        "head_object_side_effect",
+        "get_object_side_effect",
+        "get_object_return_value",
+        "expected_result",
+        "expected_exception",
+    ),
+    [
+        # Test case 1: The bootstrap file exists, content is in the new format, and contains all required features
```

Comment:
Rather than describing the test with a comment, use the test id.
See example here: https://github.com/aws/aws-parallelcluster/blob/c84baa2955744e418eccc42c93c339131ef36a06/cli/tests/pcluster/aws/test_ec2.py#L168

Reply:
Done, thanks for knowledge sharing.